conf/init/service/batch/blobuploadcleanupworker/run

```diff
@@ -1,10 +1,15 @@
 #! /bin/bash
 
+while [ -z "$(netstat -tln | grep 9091)" ]; do
+  echo 'Waiting for pushgateway to start...'
+  sleep 1
+done
+
```

Comment:
None of these `run` files are used any longer (tech debt to remove). 

If there is a desire to wait on this service, the pattern would probably be to do it in `quay-entrypoint.sh`. You'd want to check to confirm that this service was not disabled at container start, then call `supervisorctl start prometheus-pushgateway`, then wait as you are here. Then proceed to calling the full `supervisorctl` as exists currently.

Hope that helps!

Reply:
Thanks a bunch! Should do us a favor and remove these files in a separate commit or PR?

Reply:
Yes, subsequent code clean up will catch these. Not related to this PR so up to you to remove here or not.